engine.py

```diff
@@ -801,10 +803,18 @@ def convert_textures(self, temp_texture_list):
 
         files_converted = []
         texture_list = []
+        futures = []
 
         if not temp_texture_list:
             return
 
+        if self.rm.threads > 0:
+            num_threads = self.rm.threads
+        else:
+            num_threads = cpu_count() + self.rm.threads
+
+        pool = ThreadPoolExecutor(num_threads)
```

Comment:
Or even more explicitly.  If number of textures > number of cores, it appears that txmake -t:1 uses pretty much 100% of one cpu core.  So in that case you could just do that and not worry about oversubscribing the threads.